packages/calcite-components/src/components/dropdown/dropdown.tsx

```diff
@@ -374,27 +374,32 @@ export class Dropdown
     this.closeCalciteDropdown();
   }
 
+  private getTraversableItems(): HTMLCalciteDropdownItemElement[] {
+    return this.items.filter((item) => !item.disabled);
```

Comment:
could also be beneficial to check for any `item.hidden` items. I could see this as some sort of dom utility potentially.

Reply:
Agreed. I'll add the `hidden` check here and will look into extracting a util in a separate PR. I think a few keyboard navigable components could benefit from this. 🎉